frontend/src/hooks/queryBuilder/useGetExplorerQueryRange.ts

```diff
@@ -32,11 +33,11 @@ export const useGetExplorerQueryRange = (
 	const isEnabled = useMemo(() => {
 		if (!options) return isEnabledQuery;
 		if (typeof options.enabled === 'boolean') {
-			return isEnabledQuery && options.enabled;
+			return !isDependentOnQB || (isEnabledQuery && options.enabled);
```

Comment:
This would fail if options.enabled is conditionally false.

Reply:
Updated condition